README.md

```diff
@@ -594,6 +594,11 @@ css(styles2.globals);
 
 It isn't determinate whether divs will be red or blue.
 
+## Minify class names
+
+Minify class names by setting the environment variable `process.env.APHRODITE_KEYS` 
```

Comment:
Is there a reason we want to add an additional env variable, instead of just using `process.env.NODE_ENV === "production"`?